docs/book/src/introduction.md

```diff
@@ -0,0 +1 @@
+{{#include ../../../README.md:2:}}
```

Comment:
What does the 2 mean?

Reply:
Ignore the first two lines, so we don't get

```
# Cluster API

# Cluster API
```
on the front page.